operators/self-node-remediation/0.11.0/manifests/self-node-remediation.clusterserviceversion.yaml

```diff
@@ -0,0 +1,487 @@
+                      - --secure-listen-address=0.0.0.0:8443
+                      - --http2-disable
+                      - --upstream=http://127.0.0.1:8080/
+                      - --logtostderr=true
+                      - --v=0
+                    image: quay.io/brancz/kube-rbac-proxy:v0.15.0
+                    name: kube-rbac-proxy
+                    ports:
+                      - containerPort: 8443
+                        name: https
+                    resources:
+                      limits:
+                        cpu: 500m
+                        memory: 128Mi
+                      requests:
+                        cpu: 5m
+                        memory: 64Mi
+                    securityContext:
+                      allowPrivilegeEscalation: false
+                      capabilities:
+                        drop:
+                          - ALL
+                priorityClassName: system-cluster-critical
+                securityContext:
+                  runAsNonRoot: false
+                serviceAccountName: self-node-remediation-controller-manager
+                terminationGracePeriodSeconds: 10
+                topologySpreadConstraints:
+                  - labelSelector:
+                      matchLabels:
+                        control-plane: controller-manager
+                        self-node-remediation-operator: ""
+                    maxSkew: 1
+                    topologyKey: kubernetes.io/hostname
+                    whenUnsatisfiable: DoNotSchedule
+      permissions:
+        - rules:
+            - apiGroups:
+                - ""
+                - coordination.k8s.io
+              resources:
+                - configmaps
+                - leases
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - update
+                - patch
+                - delete
+            - apiGroups:
+                - ""
+              resources:
+                - events
+              verbs:
+                - create
+                - patch
+          serviceAccountName: self-node-remediation-controller-manager
+    strategy: deployment
+  installModes:
+    - supported: false
+      type: OwnNamespace
+    - supported: false
+      type: SingleNamespace
+    - supported: false
+      type: MultiNamespace
+    - supported: true
+      type: AllNamespaces
+  keywords:
+    - fencing
+    - remediation
+    - auto-healing
+    - recovery
+    - high-availability
+    - baremetal
+    - SNR
+  links:
+    - name: Self Node Remediation
+      url: https://medik8s.io
+    - name: Source Code
+      url: https://github.com/medik8s/self-node-remediation
+  maintainers:
+    - email: medik8s@googlegroups.com
+      name: Medik8s Team
+  maturity: alpha
+  minKubeVersion: 1.25.0
+  provider:
+    name: Medik8s
+    url: https://www.medik8s.io/
+  version: 0.11.0
+  replaces: self-node-remediation.v0.10.0
```

Comment:
It failed on `check_replaces_availability` even though v0.10.0 exists at https://github.com/redhat-openshift-ecosystem/community-operators-prod/tree/main/operators/self-node-remediation/0.10.0